slides/workshop_rap_for_managers_basics.qmd

```diff
@@ -309,15 +326,13 @@ This is so we always have an accurate picture of RAP in the department to share
 - <span style="color:#f67400;">**Myth**</span>: **If we move to RAP best practice, we'll have to rewrite all our SQL code into R**
 - <span style="color:#f67400;">**Reality**</span>: RAP is about using the best tool for the job. If you're using SQL and it's working for you, that's great, and there's no pressure to rewrite code if it's not necessary. We especially recommend using SQL for any heavy processing work with large datasets. 
 
-<br>
 
 - <span style="color:#f67400;">**Myth**</span>: **RAP is complicated and hugely time consuming**
-- <span style="color:#f67400;">**Reality**</span>: Gradual, small changes towards best practice all add up - you don't have to change everything all at once!
+- <span style="color:#f67400;">**Reality**</span>: Gradual, small changes towards best practice all add up - you don't have to change everything all at once! You have probably made much more progress towards best practice than you realise
```

Comment:
needs an 'already' or something to help frame it more as they're not starting from 0 here